participant-manager/src/app/about/about.component.scss

```diff
@@ -0,0 +1,10 @@
+.para_text {
```

Comment:
Should this have copyright header?

Reply:
Hi @jkrogers18,  we include copyright headers in only SVG images, not in HTML and CSS files.